docs/usage/configuration/config_documentation.md

```diff
@@ -2570,7 +2570,49 @@ Example configuration:
 ```yaml
 nonrefreshable_access_token_lifetime: 24h
 ```
+---
+### `ui_auth`
+
+The amount of time to allow a user-interactive authentication session to be active.
 
+This defaults to 0, meaning the user is queried for their credentials
+before every action, but this can be overridden to allow a single
+validation to be re-used.  This weakens the protections afforded by
+the user-interactive authentication process, by allowing for multiple
+(and potentially different) operations to use the same validation session.
+
+This is ignored for potentially "dangerous" operations (including
+deactivating an account, modifying an account password, adding a 3PID,
+and minting additional login tokens).
+
+Use the `session_timeout` sub-option here to change the time allowed for credential validation.
+
+Example configuration:
+```yaml
+ui_auth:
+    session_timeout: "15s"
+```
+---
+### `login_via_existing_session`
+
+Matrix supports the ability of an existing session to mint a login token for
+another client.
+
+Synapse disables this by default as it has security ramifications.
```

Comment:
Where can we read more about these security ramifications (can we link it)? What is this hinting at?

Reply:
It should probably be spelled out more, but allowing an existing session to login new sessions is a pretty straightforward security concern. As the [MSC says](https://github.com/matrix-org/matrix-spec-proposals/blob/hughns/login-token-requests/proposals/3882-login-token-request.md#security-considerations):

> A malicious client could use the mechanism to spawn more than one session.

Reply:
If you have access to someone's computer for a short period of time you may be able to create your own session that you transfer to a (physical) device you control or such.

Reply:
But the default setting of `login_via_existing_session`.`require_ui_auth` means that UIA is required before a token is returned. So, the account takeover is limited that way.

As such, I would argue that it could be enabled safely by default.

Reply:
> As such, I would argue that `login_via_existing_session`.`enabled` could be true by default.

I don't have strong feelings about it, I just defaulted to the safe way. I'd be fine changing this if you think it is fine to.